src/Components/Artwork/index.tsx

```diff
@@ -111,11 +111,9 @@ export class Artwork extends React.Component<ArtworkProps, ArtworkState> {
       <Container onClick={this.onSelected}>
         <ImageContainer>
           <Image src={artwork.image.url} />
-          <div className={overlayClasses}>
-            {Overlay && <Overlay selected={this.state.isSelected} />}
-          </div>
+          <div className={overlayClasses}>{Overlay && <Overlay selected={this.state.isSelected} />}</div>
         </ImageContainer>
-        <ArtworkMetadata artwork={artwork} />
+        <ArtworkMetadata extended={this.props.extended} artwork={artwork} />
```

Comment:
This was basically a/the bug. The loyalty inquiry artworks were using this component with false, but that was never being passed in to `ArtworkMetadata`. So, with this fix its safe to change the default in the `Metadata` component.